region/src/main/java/org/motechproject/nms/region/domain/State.java

```diff
@@ -80,22 +68,14 @@ public void setCode(Long code) {
         this.code = code;
     }
 
-    public List<District> getDistricts() {
+    public Set<District> getDistricts() {
```

Comment:
With a list there is a column added on the many side that is no longer used when the variable is a set.  So ideally that column would be dropped, but the schema should be compatible.